great_tables/_boxhead.py

```diff
@@ -5,14 +5,14 @@
 from ._locations import resolve_cols_c
 from ._utils import _assert_list_is_subset
 from ._tbl_data import SelectExpr
-from ._text import Text
+from ._text import BaseText
```

Comment:
Please don't put sweeping refactor style changes in feature PRs

Reply:
Now fixed up during code review.